_data/simple-icons.json

```
@@ -7439,6 +7439,11 @@
             "source": "https://openwrt.org/docs/guide-graphic-designer/openwrt-logo",
             "guidelines": "https://openwrt.org/docs/guide-graphic-designer/openwrt-logo"
         },
+        {
+            "title": "OpenZeppelin",
+            "hex": "4E5EE4",
+            "source": "https://openzeppelin.com/"
```

Since the brand guidelines are included in the logo kit ZIP archive, I'd add the `guidelines` field, too:

```suggestion
            "source": "https://openzeppelin.com/",
            "guidelines": "https://www.openzeppelin.com/"
```


That said, I think it would be more informative to have a dedicated page for the brand on your website, e.g. https://www.openzeppelin.com/brand/.